cmake/third_party/FindCUDNN.cmake

```
@@ -13,7 +13,7 @@ include(FindPackageHandleStandardArgs)
 
 set(CUDNN_ROOT_DIR "" CACHE PATH "Folder contains NVIDIA cuDNN")
 
-option(CUDNN_STATIC "Look for static cuDNN" ON)
+option(CUDNN_STATIC "Look for static cuDNN" OFF)
```

```suggestion
set(CUDNN_STATIC ON CACHE BOOL "" FORCE)
```
改成这个在本地试试看，我看log因为有cmake缓存，还是用的静态库（用动态库的好像失败了https://github.com/Oneflow-Inc/oneflow/actions/runs/741110557）